### PR TITLE
Increase max grpc message size

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: cmd/crio/main.go [hunk 1]
```diff
@@ -454,7 +454,12 @@ func main() {
 			logrus.Fatalf("failed to listen: %v", err)
 		}
 
-		s := grpc.NewServer()
+		// The default grpc msg size is 4MB.
+		// We bump it up to 16MB for allowing more containers.
+		maxMsgSize := 16 * 1024 * 1024
+		s := grpc.NewServer(
+			grpc.MaxMsgSize(maxMsgSize),
+		)
```
Comment (TomSweeneyRedHat) [inline cmd/crio/main.go]: is this anything that someone would want to toggle via crio.conf?

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but tests aren't hip

File: cmd/crio/main.go (hunk 1 above)
Comment (mrunalp) [reply cmd/crio/main.go]: We added this in newer branches with crio.conf tunables. We don't want to support that for 1.9.

**Comment by giuseppe** [comment]
/test e2e_fedora
/test e2e_rhel

LGTM

**Pull request merged**